Minor changes to 0.11 docs (#583)
kuzudb/kuzu-docs: src/content/docs/cypher/transaction.md

Kuzu implements a transaction management subsystem that is atomic, durable and supports serializability.
Satisfying these properties makes Kuzu ACID-compliant, as per database terminology.

> Every query, data manipulation command, DDL (i.e., new node/rel table schema definitions), or `COPY FROM` command to Kuzu is part of a transaction. Therefore, they exhibit all-or-nothing behavior; after these commands (or a set of them) execute and are committed successfully, you are guaranteed that all changes will persist in their entirety. If the commands do not execute successfully or are rolled back, you are guaranteed that none of the changes will persist.

Reviewer: ```suggestion
Every query, data manipulation command, DDL (i.e., new node/rel table schema definitions), or `COPY FROM` command to Kuzu is part of a transaction. Therefore, they exhibit all-or-nothing behavior. After one or more of these commands are executed and committed successfully, you are guaranteed that all changes will persist in their entirety. If the commands do not execute successfully or are rolled back, you are guaranteed that none of the changes will persist.
```

Author: updated
